src/bmark/bmark_solvers.jl

```diff
@@ -1,4 +1,7 @@
-export bmark_solvers, profile_solvers, bmark_and_profile
+export bmark_solvers, profile_solvers, profile_solvers_by_time,
+       profile_solvers_by_iterations, profile_solvers_by_evaluations,
+       bmark_and_profile, bmark_and_profile_by_time,
+       bmark_and_profile_by_iterations, bmark_and_profile_by_evaluations
```

Comment:
I'm doing that here, albeit less clean: `cost=stat->stat.elapsed_time`. For the evaluations, I have to do `stat->sum(getfield(stat.eval, x) for x in fieldnames(NLPModels.Counters))`.
The `*_by_*` functions are specific `cost` choices.